### PR TITLE
httpclient 4.5.13

### PR DESCRIPTION
**Pull request by wcekan**
## Description
Avoid CVE-2020-13956
```
httpclient-4.5.3.jar (pkg:maven/org.apache.httpcomponents/httpclient@4.5.3, cpe:2.3:a:apache:httpclient:4.5.3:*:*:*:*:*:*:*) : CVE-2020-13956
```

## Motivation and Context
Security

## How Has This Been Tested?
Unit testing

## License
I confirm that this contribution is made under an Apache 2.0 license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: elide-datastore/elide-datastore-aggregation/pom.xml [hunk 1]
```diff
@@ -104,6 +104,12 @@
             </exclusions>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
```
Comment (aklish) [inline elide-datastore/elide-datastore-aggregation/pom.xml:108]: What requires this?

Comment (wcekan) [reply elide-datastore/elide-datastore-aggregation/pom.xml:108]: avatica-core

Comment (wcekan) [reply elide-datastore/elide-datastore-aggregation/pom.xml:112]: changed scope to runtime to match the dependencies of calcite

Comment (aklish) [reply elide-datastore/elide-datastore-aggregation/pom.xml:102]: We don't use calcite to make HTTP connections though.  Is there any reason we can't simply exclude it?

Comment (wcekan) [reply elide-datastore/elide-datastore-aggregation/pom.xml:102]: No.  If we mention `calcite` without any mention of `httpclient` the downstream will pull in the `calcite` old version.  They have the option to exclude it, but sadly `maven` does not let us exclude on their behalf.

**Review by aklish — APPROVED** [review]

File: elide-datastore/elide-datastore-aggregation/pom.xml (hunk 1 above)
Comment (aklish) [reply elide-datastore/elide-datastore-aggregation/pom.xml:102]: Can we add a comment:

Avoid CVE-2020-13956

**Review by aklish — CHANGES_REQUESTED** [review]

**Review by aklish — APPROVED** [review]

**Pull request merged**